deploy-service/common/src/main/java/com/pinterest/clusterservice/common/ClusterConstants.java

```diff
@@ -0,0 +1,23 @@
+/**
+ * Copyright 2016 Pinterest, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.pinterest.clusterservice.common;
+
+
+public class ClusterConstants {
```

Comment:
I suggest to move these constants to AwsVmManager, they are AWS specific, also usually we like to put constants to where it belongs to.